lib/suspenders/app_builder.rb

```diff
@@ -271,8 +271,8 @@ def setup_foreman
 
     def setup_stylesheets
       remove_file 'app/assets/stylesheets/application.css'
-      copy_file 'application.css.scss',
-        'app/assets/stylesheets/application.css.scss'
+      copy_file 'application.scss',
+        'app/assets/stylesheets/application.scss'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.